.github/release-drafter.yml

```diff
@@ -0,0 +1,33 @@
+name-template: Siddhi IO HTTP $NEXT_PATCH_VERSION Release
+tag-template: v$NEXT_PATCH_VERSION
+change-template: '* $TITLE (#$NUMBER)'
+
+categories:
+  - title: 'Features and Improvements'
+    labels:
+      - type/new-feature
+      - type/improvement
+  - title: 'Bug Fixes'
+    labels:
+      - type/bug
+
+template: |
+  > This Release is compatible with Siddhi 5.x.x
```

Comment:
Is it possible to add the correct Siddhi version automatically?

Reply:
No, it's not possible as of now. We have to handle it manually.